cmd/soroban-cli/src/commands/network/container/logs.rs

```diff
@@ -0,0 +1,52 @@
+use chrono::offset::Local;
+use futures_util::TryStreamExt;
+
+use crate::commands::network::container::shared::{
+    connect_to_docker, Error as ConnectionError, Network, DOCKER_HOST_HELP,
+};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    ConnectionError(#[from] ConnectionError),
+
+    #[error("⛔ ️Failed to tail container: {0}")]
+    TailContainerError(#[from] bollard::errors::Error),
+}
+
+#[derive(Debug, clap::Parser, Clone)]
+pub struct Cmd {
+    /// Network to tail
+    pub network: Network,
+
+    #[arg(short = 'd', long, help = DOCKER_HOST_HELP, env = "DOCKER_HOST")]
+    pub docker_host: Option<String>,
+}
+
+impl Cmd {
+    pub async fn run(&self) -> Result<(), Error> {
+        let container_name = format!("stellar-{}", self.network);
+        println!("ℹ️  Tailing logs for {container_name}");
+        let docker = connect_to_docker(&self.docker_host).await?;
+        let logs_stream = &mut docker.logs(
+            &container_name,
+            Some(bollard::container::LogsOptions {
+                follow: true,
+                stdout: true,
+                stderr: true,
+                tail: "all",
+                ..Default::default()
+            }),
+        );
+
+        while let Some(log) = logs_stream.try_next().await? {
+            print!(
+                "{}: {} {}",
+                container_name,
+                Local::now().format("%Y-%m-%d %H:%M:%S"),
```

Comment:
I'd lean towards not augmenting the logs that we output. If the YMD is useful, we could add them directly to quickstart.

I'd lean towards leaving the container name out too. Screen real estate is valuable and it adds additional width. As far as I can tell we'll only output logs for a single instance at a time, so we don't have to distinguish between them.

Let me know if I'm missing something.

Reply:
That is a good point. I was trying to match the output I was seeing in Docker Desktop.. but it looks like they are adding YMD themselves, which we don't see when just attaching to a container. I'll remove the YMD, and the container name!

Reply:
Nice! I've add a new issue for this: https://github.com/stellar/stellar-cli/issues/1365